client/components/navbar/navbar.service.js

```diff
@@ -383,6 +384,17 @@ class ManagerNavbarService {
         }),
         mustBeKept: _.has(currentSubsidiaryURLs, 'support_contact'),
       },
+      {
+        title: `${this.$translate.instant('common_menu_support_chatbot')} <sup class="oui-color-california">OVH Chat</sup>`,
+        click: () => {
+          this.$rootScope.$broadcast('ovh-chatbot:open');
```

Comment:
As you're in a service you can't emit on `$scope` but we should try to avoid this as much as possible